DOC: prepare 1.13.0 release notes (#19880)
scipy/scipy: doc/source/release/1.13.0-notes.rst

> `scipy.interpolate` improvements

Reviewer: ```suggestion
### scipy.integrate improvements
- The `terminal` attribute of event functions passed to `scipy.integrate.solve_ivp` may now be a positive integer: integration stops after `terminal` events have occurred.

`scipy.interpolate` improvements
```

Author: Yeah, I've already got this one locally. I guess we'll see how things compare when I push up in a bit.

`scipy.special` improvements
============================
- The ``AMOS`` library was rewritten in C, from its original Fortran
  implementation.
- ``cdflib`` has been rewritten in Cython, from its original Fortran
  implementation.

`scipy.stats` improvements
==========================

> - `scipy.stats.rankdata` has been vectorized, improving its performance and the
>   performance of hypothesis tests that depend on it.
> - ``stats.mannwhitneyu`` should now be faster due to a vectorized statistic
>   calculation.
> - `scipy.stats.mood` now has ``nan_policy`` and ``keepdims`` support.
> - `scipy.stats.shapiro`, `scipy.stats.normaltest`, `scipy.stats.skewtest`,
>   `scipy.stats.kurtosistest`, and `scipy.stats.kstest` have gained ``axis``,
>   ``nan_policy`` and ``keepdims`` support.
> - `scipy.stats.boxcox_normmax` has gained a ``ymax`` parameter to allow user
>   specification of the maximum value of the transformed data.

Reviewer: ```suggestion
- `scipy.stats.wasserstein_distance_nd` was introduced to compute the Wasserstein-1 distance between two N-D discrete distributions.
- Vectorized calculations with `scipy.stats.wilcoxon`, `scipy.stats.mannwhitneyu`, and `scipy.stats.rankdata` are faster.
- The `method` parameter of `scipy.stats.wilcoxon` and `scipy.stats.mannwhitneyu` now accepts instances of `scipy.stats.PermutationMethod` for computing exact p-values in the presents of ties.
- `scipy.stats.boxcox_normmax` now accepts `ymax` parameter, which allows users to specify the maximum allowable magnitude of the transformed data when ``method='mle'``.
- The ``fit`` methods of  `scipy.stats.gamma` (with `method='mm'`) and `scipy.stats.loglaplace` are faster and more reliable.
- Moment calculations of `scipy.stats.powerlaw` are faster and more accurate.
- The `statistic` parameter of `scipy.stats.goodness_of_fit` now accepts custom callables.
- Support for `axis`, `keepdims`, and `nan_policy` keyword arguments has been added or improved for the following functions: `scipy.stats.f_oneway`, `scipy.stats.alexandergovern`, scipy.stats.shapiro`, `scipy.stats.normaltest`, scipy.stats.skewtest`, scipy.stats.kurtosistest`, scipy.stats.friedmanchisquare`, scipy.stats.brunnermunzel`, and `scipy.stats.mood`.
```

> - `scipy.stats.invwishart` ``rvs`` and ``logpdf`` are now faster.

Reviewer: ```suggestion
```

> Hypothesis Tests

Reviewer: These categories can be removed when you're editing locally.
